cita-executor/core/src/contracts/solc/chain_manager.rs

```diff
@@ -35,64 +36,72 @@ lazy_static! {
 
 pub struct ChainManagement;
 
+use cita_vm::evm::DataProvider;
+
 impl ChainManagement {
-    pub fn ext_chain_id(ext: &mut Ext, gas: &U256, sender: &Address) -> Option<(U256, U256)> {
+    pub fn ext_chain_id(
+        _ext: &mut DataProvider,
+        _gas: &U256,
+        _sender: &Address,
+    ) -> Option<(U256, U256)> {
         trace!("call system contract ChainManagement.ext_chain_id()");
-        let contract = &*CONTRACT_ADDRESS;
+        let _contract = &*CONTRACT_ADDRESS;
         let tx_data = CHAIN_ID_ENCODED.to_vec();
-        let data = &tx_data.as_slice();
-        let mut output = Vec::<u8>::new();
-        match ext.call(
-            gas,
-            sender,
-            contract,
-            None,
-            data,
-            contract,
-            &mut output,
-            CallType::Call,
-        ) {
-            MessageCallResult::Success(gas_left, return_data) => {
-                decode_tools::to_u256(&*return_data).map(|x| (gas_left, x))
-            }
-            MessageCallResult::Reverted(..) | MessageCallResult::Failed => None,
-        }
+        let _data = &tx_data.as_slice();
+        let _output = Vec::<u8>::new();
+        // match ext.call(
```

Comment:
You need to implement this logic.

Reply:
This Logic is similar to `eth_call`. I will implement the logic of `eth_call`, and you can reference it.